src/SFML/Window/GlContext.cpp

```diff
@@ -473,6 +473,19 @@ void GlContext::checkSettings(const ContextSettings& requestedSettings)
 {
     // Perform checks to inform the user if they are getting a context they might not have expected
 
+    // Detect any known non-accelerated implementations and warn
+    const char* vendorName = reinterpret_cast<const char*>(glGetString(GL_VENDOR));
+    const char* rendererName = reinterpret_cast<const char*>(glGetString(GL_RENDERER));
+
+    if (vendorName && rendererName)
+    {
+        if (!std::strcmp(vendorName, "Microsoft Corporation") && !std::strcmp(rendererName, "GDI Generic"))
```

Comment:
Fixed. Those other locations will be cleaned up in a bigger refactor of all the GLLoader code in 2.4.